### ISSUE TITLE
Unable to Retrieve the password for gSMA Account

### ISSUE THREAD

**Issue by SujeshSasidharan** [Ryan2065/gMSACredentialModule#1]
Hi All,
I did follow the same steps as mentioned , but I am not able to retrieve the gSMA Password and it is coming as blank. Could you please let me know how I Can check where it is going wrong. what kind of permission does the user needs to retrieve the password.

**Comment by jpscripter** [comment]
Typically, that means you are not in the AD Group that can manage the GMSA Account's Password. If you just added your user to this group, you will need to relogin or kill your kerberos tickets with the klist command.

**Comment by FGNicky** [comment]
When you say not in the AD group, do you also have to be a member of this group (from your example code)?
-PrincipalsAllowedToRetrieveManagedPassword $ADGroupName
or is it a different one?

**Comment by jpscripter** [comment]
Yep, the AD Group that you usually put the computer accounts into is the one that your account needs to be in. I dont have an example but make sure your AD Account in the same account that you have in the $ADGroupName variable.
